src/platform/packages/shared/kbn-unified-tabs/src/components/tab/tab.tsx

```diff
@@ -210,6 +210,7 @@ export const Tab: React.FC<TabProps> = (props) => {
                   // Truncation width must be equal to max tab width minus padding
                   width={tabsSizeConfig.regularTabMaxWidth - euiTheme.base}
                   truncation="middle"
+                  title=""
```

Comment:
Huh, interesting this was the cause of the title. Had no idea it was built in to the component.